core/src/test/resources/csv-inputs/forum.csv

```diff
@@ -0,0 +1,2 @@
+id:ID(Forum)|title:STRING|creationDate:LONG
+0|Wall of Mahinda Perera|20100214153220447
```

Comment:
Is the file "forum.csv" used? I can do a search and find "personsWithoutIdField.csv" but not this one

Reply:
This is leftover from having built this PR on top of another. It shouldn't be there anymore. This file is generated automatically by some tests.